INFO 2024-06-09 01:22:36,044 autoreload 7524 3656 Watching for file changes with StatReloader
INFO 2024-06-09 01:22:41,854 autoreload 13932 12240 Watching for file changes with StatReloader
INFO 2024-06-09 01:22:44,554 basehttp 13932 11260 "GET / HTTP/1.1" 200 13
WARNING 2024-06-09 01:22:44,671 log 13932 11260 Not Found: /favicon.ico
WARNING 2024-06-09 01:22:44,671 basehttp 13932 11260 "GET /favicon.ico HTTP/1.1" 404 2546
INFO 2024-06-09 01:22:50,433 basehttp 13932 11260 "GET /game HTTP/1.1" 301 0
INFO 2024-06-09 01:22:50,441 basehttp 13932 12660 "GET /game/ HTTP/1.1" 200 56
INFO 2024-06-09 01:22:55,405 basehttp 13932 12660 "GET /game/coin HTTP/1.1" 301 0
INFO 2024-06-09 01:22:55,412 views 13932 12876 Index page accessed
INFO 2024-06-09 01:22:55,412 basehttp 13932 12876 "GET /game/coin/ HTTP/1.1" 200 38
INFO 2024-06-09 01:23:00,558 views 13932 12876 Index page accessed
INFO 2024-06-09 01:23:00,559 basehttp 13932 12876 "GET /game/random_number/ HTTP/1.1" 200 32
INFO 2024-06-09 01:23:05,030 basehttp 13932 12876 "GET /game/ HTTP/1.1" 200 56
INFO 2024-06-09 01:28:11,997 autoreload 13932 12240 C:\GB\Django\practika\myproject\myproject\settings.py changed, reloading.
INFO 2024-06-09 01:28:12,505 autoreload 2648 5420 Watching for file changes with StatReloader
INFO 2024-06-09 01:28:17,686 autoreload 2256 10236 Watching for file changes with StatReloader
INFO 2024-06-09 01:28:28,420 basehttp 2256 3484 "GET / HTTP/1.1" 200 13
INFO 2024-06-09 01:28:39,329 basehttp 2256 3484 "GET /about HTTP/1.1" 301 0
INFO 2024-06-09 01:28:39,337 basehttp 2256 9724 "GET /about/ HTTP/1.1" 200 8
WARNING 2024-06-09 01:28:43,587 log 2256 9724 Not Found: /abousdfsdf
WARNING 2024-06-09 01:28:43,594 basehttp 2256 9724 "GET /abousdfsdf HTTP/1.1" 404 2543
INFO 2024-06-09 01:28:47,783 basehttp 2256 9724 "GET / HTTP/1.1" 200 13
INFO 2024-06-09 01:29:40,181 autoreload 2256 10236 C:\GB\Django\practika\myproject\myproject\settings.py changed, reloading.
INFO 2024-06-09 01:29:43,910 autoreload 8324 4380 Watching for file changes with StatReloader
INFO 2024-06-09 01:29:47,577 basehttp 8324 14072 "GET / HTTP/1.1" 200 13
INFO 2024-06-09 01:29:51,804 basehttp 8324 14072 "GET /about/ HTTP/1.1" 200 8
WARNING 2024-06-09 01:29:55,513 log 8324 14072 Not Found: /abofghfgh
WARNING 2024-06-09 01:29:55,513 basehttp 8324 14072 "GET /abofghfgh HTTP/1.1" 404 2540
INFO 2024-06-09 01:29:58,518 basehttp 8324 14072 "GET / HTTP/1.1" 200 13
INFO 2024-06-09 01:32:58,222 autoreload 8324 4380 C:\GB\Django\practika\myproject\myproject\settings.py changed, reloading.
INFO 2024-06-09 01:32:58,730 autoreload 3484 16660 Watching for file changes with StatReloader
INFO 2024-06-09 01:33:02,774 autoreload 1980 2728 Watching for file changes with StatReloader
INFO 2024-06-09 01:33:09,825 basehttp 1980 11516 "GET / HTTP/1.1" 200 13
INFO 2024-06-09 01:33:13,326 basehttp 1980 11516 "GET /about/ HTTP/1.1" 200 8
WARNING 2024-06-09 01:33:16,391 log 1980 11516 Not Found: /aboujkljkl
WARNING 2024-06-09 01:33:16,391 basehttp 1980 11516 "GET /aboujkljkl HTTP/1.1" 404 2543
INFO 2024-06-09 01:33:19,120 basehttp 1980 11516 "GET / HTTP/1.1" 200 13
INFO 2024-06-09 01:34:30,286 autoreload 1980 2728 C:\GB\Django\practika\myproject\myapp1\views.py changed, reloading.
INFO 2024-06-09 01:34:30,755 autoreload 11388 16880 Watching for file changes with StatReloader
INFO 2024-06-09 01:34:33,746 autoreload 8588 14740 Watching for file changes with StatReloader
INFO 2024-06-09 01:34:37,253 basehttp 8588 15624 "GET / HTTP/1.1" 200 13
INFO 2024-06-09 01:34:40,972 basehttp 8588 15624 "GET /about/ HTTP/1.1" 200 8
WARNING 2024-06-09 01:34:44,108 log 8588 15624 Not Found: /abouthjkh'
WARNING 2024-06-09 01:34:44,116 basehttp 8588 15624 "GET /abouthjkh' HTTP/1.1" 404 2558
INFO 2024-06-09 01:34:48,668 basehttp 8588 15624 "GET / HTTP/1.1" 200 13
INFO 2024-06-09 01:46:17,156 autoreload 8588 14740 C:\GB\Django\practika\myproject\myapp1\views.py changed, reloading.
INFO 2024-06-09 01:46:17,857 autoreload 8760 3212 Watching for file changes with StatReloader
INFO 2024-06-09 01:46:45,704 autoreload 8760 3212 C:\GB\Django\practika\myproject\myapp1\views.py changed, reloading.
INFO 2024-06-09 01:46:46,430 autoreload 13132 13896 Watching for file changes with StatReloader
INFO 2024-06-09 01:48:08,031 autoreload 13132 13896 C:\GB\Django\practika\myproject\myapp1\views.py changed, reloading.
INFO 2024-06-09 01:48:08,763 autoreload 6944 12692 Watching for file changes with StatReloader
INFO 2024-06-09 01:48:32,261 autoreload 6944 12692 C:\GB\Django\practika\myproject\myapp1\views.py changed, reloading.
INFO 2024-06-09 01:48:32,931 autoreload 10884 3528 Watching for file changes with StatReloader
INFO 2024-06-09 01:51:43,899 autoreload 5972 6300 Watching for file changes with StatReloader
INFO 2024-06-09 01:57:58,821 autoreload 7784 11760 Watching for file changes with StatReloader
ERROR 2024-06-09 01:58:06,875 log 7784 16168 Internal Server Error: /
Traceback (most recent call last):
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\myproject\myapp1\views.py", line 9, in index
    response = render_to_string('myapp1/main.html')
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 481, in parse
    raise self.error(token, e)
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 479, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 597, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 698, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '()' from 'super()'
ERROR 2024-06-09 01:58:06,887 basehttp 7784 16168 "GET / HTTP/1.1" 500 167741
INFO 2024-06-09 01:59:48,802 autoreload 1124 7240 Watching for file changes with StatReloader
INFO 2024-06-09 01:59:51,279 basehttp 1124 16216 "GET / HTTP/1.1" 200 142
INFO 2024-06-09 02:01:51,906 autoreload 1124 7240 C:\GB\Django\practika\myproject\myproject\settings.py changed, reloading.
INFO 2024-06-09 02:01:55,405 autoreload 2176 12152 Watching for file changes with StatReloader
INFO 2024-06-09 02:01:59,924 basehttp 2176 14300 "GET / HTTP/1.1" 200 142
INFO 2024-06-09 02:02:00,414 basehttp 2176 14300 "GET / HTTP/1.1" 200 142
INFO 2024-06-09 02:02:02,475 basehttp 2176 14300 "GET / HTTP/1.1" 200 142
INFO 2024-06-09 02:02:02,963 basehttp 2176 14300 "GET / HTTP/1.1" 200 142
INFO 2024-06-09 02:02:19,911 basehttp 2176 14300 "GET / HTTP/1.1" 200 142
INFO 2024-06-09 02:02:20,781 autoreload 2176 12152 C:\GB\Django\practika\myproject\myapp1\views.py changed, reloading.
INFO 2024-06-09 02:02:21,344 autoreload 14788 7516 Watching for file changes with StatReloader
ERROR 2024-06-09 02:02:24,874 log 14788 10504 Internal Server Error: /
Traceback (most recent call last):
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\myproject\myapp1\views.py", line 9, in index
    response = render_to_string('myapp1/main.html')
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    raise self.error(token, e)
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 508, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 481, in parse
    raise self.error(token, e)
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 479, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 597, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 698, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '()' from 'super()'
ERROR 2024-06-09 02:02:24,897 basehttp 14788 10504 "GET / HTTP/1.1" 500 166358
INFO 2024-06-09 02:06:33,533 autoreload 10004 15608 Watching for file changes with StatReloader
INFO 2024-06-09 02:06:47,615 autoreload 13040 14240 Watching for file changes with StatReloader
ERROR 2024-06-09 02:06:50,254 log 13040 13880 Internal Server Error: /
Traceback (most recent call last):
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\myproject\myapp1\views.py", line 9, in index
    response = render_to_string('myapp1/main1.html')
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 481, in parse
    raise self.error(token, e)
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 479, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 597, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 698, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '('index')' from 'url_for('index')'
ERROR 2024-06-09 02:06:50,264 basehttp 13040 13880 "GET / HTTP/1.1" 500 137145
ERROR 2024-06-09 02:06:51,128 log 13040 13880 Internal Server Error: /
Traceback (most recent call last):
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\myproject\myapp1\views.py", line 9, in index
    response = render_to_string('myapp1/main1.html')
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 481, in parse
    raise self.error(token, e)
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 479, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 597, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\GB\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 698, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '('index')' from 'url_for('index')'
ERROR 2024-06-09 02:06:51,138 basehttp 13040 13880 "GET / HTTP/1.1" 500 137145
INFO 2024-06-09 02:09:26,894 autoreload 8592 9168 Watching for file changes with StatReloader
INFO 2024-06-09 02:09:29,299 basehttp 8592 16024 "GET / HTTP/1.1" 200 1002
WARNING 2024-06-09 02:09:30,916 log 8592 16024 Not Found: /main1.html
WARNING 2024-06-09 02:09:30,916 basehttp 8592 16024 "GET /main1.html HTTP/1.1" 404 2543
WARNING 2024-06-09 02:09:33,024 log 8592 16024 Not Found: /main1.html
WARNING 2024-06-09 02:09:33,024 basehttp 8592 16024 "GET /main1.html HTTP/1.1" 404 2543
WARNING 2024-06-09 02:09:35,648 log 8592 16024 Not Found: /main1.html
WARNING 2024-06-09 02:09:35,648 basehttp 8592 16024 "GET /main1.html HTTP/1.1" 404 2543
WARNING 2024-06-09 02:09:41,478 log 8592 16024 Not Found: /main1.html
WARNING 2024-06-09 02:09:41,478 basehttp 8592 16024 "GET /main1.html HTTP/1.1" 404 2543
INFO 2024-06-09 02:10:33,161 autoreload 15556 10228 Watching for file changes with StatReloader
INFO 2024-06-09 02:10:37,138 basehttp 15556 2196 "GET / HTTP/1.1" 200 994
WARNING 2024-06-09 02:10:38,136 log 15556 2196 Not Found: /main1.html
WARNING 2024-06-09 02:10:38,136 basehttp 15556 2196 "GET /main1.html HTTP/1.1" 404 2543
INFO 2024-06-09 02:11:09,972 autoreload 14024 16072 Watching for file changes with StatReloader
WARNING 2024-06-09 02:11:12,296 log 14024 11824 Not Found: /abouthjkh'
WARNING 2024-06-09 02:11:12,299 basehttp 14024 11824 "GET /abouthjkh' HTTP/1.1" 404 2558
INFO 2024-06-09 02:11:15,295 basehttp 14024 11824 "GET / HTTP/1.1" 200 982
INFO 2024-06-09 02:11:16,579 basehttp 14024 11824 "GET / HTTP/1.1" 200 982
INFO 2024-06-09 02:11:17,290 basehttp 14024 11824 "GET / HTTP/1.1" 200 982
INFO 2024-06-09 02:11:17,667 basehttp 14024 11824 "GET / HTTP/1.1" 200 982
INFO 2024-06-09 02:11:18,015 basehttp 14024 11824 "GET / HTTP/1.1" 200 982
INFO 2024-06-09 02:11:18,249 basehttp 14024 11824 "GET / HTTP/1.1" 200 982
INFO 2024-06-09 02:11:18,692 basehttp 14024 11824 "GET / HTTP/1.1" 200 982
INFO 2024-06-09 02:11:18,999 basehttp 14024 11824 "GET / HTTP/1.1" 200 982
INFO 2024-06-09 02:11:19,656 basehttp 14024 11824 "GET / HTTP/1.1" 200 982
INFO 2024-06-09 02:11:19,924 basehttp 14024 11824 "GET / HTTP/1.1" 200 982
INFO 2024-06-09 02:11:21,079 basehttp 14024 11824 "GET / HTTP/1.1" 200 982
INFO 2024-06-09 02:11:21,422 basehttp 14024 11824 "GET / HTTP/1.1" 200 982
INFO 2024-06-09 02:15:12,357 autoreload 14228 15248 Watching for file changes with StatReloader
INFO 2024-06-09 02:15:15,245 basehttp 14228 11860 "GET / HTTP/1.1" 200 941
INFO 2024-06-09 02:15:16,956 basehttp 14228 11860 "GET / HTTP/1.1" 200 941
INFO 2024-06-09 02:15:17,581 basehttp 14228 11860 "GET / HTTP/1.1" 200 941
INFO 2024-06-09 02:15:18,210 basehttp 14228 11860 "GET / HTTP/1.1" 200 941
INFO 2024-06-09 02:15:18,677 basehttp 14228 11860 "GET / HTTP/1.1" 200 941
INFO 2024-06-09 02:15:19,164 basehttp 14228 11860 "GET / HTTP/1.1" 200 941
INFO 2024-06-09 02:15:51,003 autoreload 1956 8740 Watching for file changes with StatReloader
INFO 2024-06-09 02:15:53,117 basehttp 1956 11316 "GET / HTTP/1.1" 200 947
INFO 2024-06-09 02:15:53,926 basehttp 1956 11316 "GET /about/ HTTP/1.1" 200 8
INFO 2024-06-09 02:16:36,049 autoreload 1956 8740 C:\GB\Django\practika\myproject\myapp1\views.py changed, reloading.
INFO 2024-06-09 02:16:38,647 autoreload 11832 1332 Watching for file changes with StatReloader
INFO 2024-06-09 02:16:40,573 basehttp 11832 3984 "GET / HTTP/1.1" 200 947
INFO 2024-06-09 02:16:41,296 basehttp 11832 3984 "GET /about/ HTTP/1.1" 200 893
INFO 2024-06-09 02:16:42,339 basehttp 11832 3984 "GET /about/ HTTP/1.1" 200 893
WARNING 2024-06-09 02:16:43,041 log 11832 3984 Not Found: /about/about/
WARNING 2024-06-09 02:16:43,041 basehttp 11832 3984 "GET /about/about/ HTTP/1.1" 404 2549
WARNING 2024-06-09 02:16:45,010 log 11832 3984 Not Found: /about/about/
WARNING 2024-06-09 02:16:45,012 basehttp 11832 3984 "GET /about/about/ HTTP/1.1" 404 2549
WARNING 2024-06-09 02:16:47,633 log 11832 3984 Not Found: /about/about/
WARNING 2024-06-09 02:16:47,633 basehttp 11832 3984 "GET /about/about/ HTTP/1.1" 404 2549
WARNING 2024-06-09 02:16:50,464 log 11832 3984 Not Found: /about/about/
WARNING 2024-06-09 02:16:50,464 basehttp 11832 3984 "GET /about/about/ HTTP/1.1" 404 2549
INFO 2024-06-09 02:17:08,043 autoreload 9200 15272 Watching for file changes with StatReloader
INFO 2024-06-09 02:17:09,776 basehttp 9200 4404 "GET /about/ HTTP/1.1" 200 887
INFO 2024-06-09 02:17:10,478 basehttp 9200 4404 "GET /about/ HTTP/1.1" 200 887
INFO 2024-06-09 02:17:11,164 basehttp 9200 4404 "GET /about/ HTTP/1.1" 200 887
INFO 2024-06-09 02:17:11,660 basehttp 9200 4404 "GET /about/ HTTP/1.1" 200 887
INFO 2024-06-09 02:17:11,862 basehttp 9200 4404 "GET /about/ HTTP/1.1" 200 887
INFO 2024-06-09 02:17:12,307 basehttp 9200 4404 "GET /about/ HTTP/1.1" 200 887
INFO 2024-06-09 02:17:12,501 basehttp 9200 4404 "GET /about/ HTTP/1.1" 200 887
INFO 2024-06-09 02:17:12,642 basehttp 9200 4404 "GET /about/ HTTP/1.1" 200 887
INFO 2024-06-09 02:17:15,033 basehttp 9200 4404 "GET / HTTP/1.1" 200 947
INFO 2024-06-09 02:17:16,837 basehttp 9200 4404 "GET /about/ HTTP/1.1" 200 887
INFO 2024-06-09 02:17:17,478 basehttp 9200 4404 "GET /about/ HTTP/1.1" 200 887
INFO 2024-06-09 02:17:19,389 basehttp 9200 4404 "GET /about/ HTTP/1.1" 200 887
INFO 2024-06-09 02:17:19,569 basehttp 9200 4404 "GET /about/ HTTP/1.1" 200 887
INFO 2024-06-09 02:17:20,028 basehttp 9200 4404 "GET /about/ HTTP/1.1" 200 887
INFO 2024-06-09 02:17:20,495 basehttp 9200 4404 "GET /about/ HTTP/1.1" 200 887
INFO 2024-06-09 02:17:20,649 basehttp 9200 4404 "GET /about/ HTTP/1.1" 200 887
INFO 2024-06-09 02:17:21,813 basehttp 9200 4404 "GET /about/ HTTP/1.1" 200 887
INFO 2024-06-09 02:17:22,739 basehttp 9200 4404 "GET /about/ HTTP/1.1" 200 887
INFO 2024-06-09 02:17:23,623 basehttp 9200 4404 "GET /about/ HTTP/1.1" 200 887
INFO 2024-06-09 02:17:42,446 autoreload 3100 13404 Watching for file changes with StatReloader
INFO 2024-06-09 02:17:45,226 basehttp 3100 13992 "GET /about/ HTTP/1.1" 200 889
INFO 2024-06-09 02:17:45,907 basehttp 3100 13992 "GET /about/ HTTP/1.1" 200 889
INFO 2024-06-09 02:17:46,301 basehttp 3100 13992 "GET /about/ HTTP/1.1" 200 889
INFO 2024-06-09 02:17:46,740 basehttp 3100 13992 "GET /about/ HTTP/1.1" 200 889
INFO 2024-06-09 02:17:46,896 basehttp 3100 13992 "GET /about/ HTTP/1.1" 200 889
INFO 2024-06-09 02:17:47,276 basehttp 3100 13992 "GET /about/ HTTP/1.1" 200 889
INFO 2024-06-09 02:17:47,455 basehttp 3100 13992 "GET /about/ HTTP/1.1" 200 889
INFO 2024-06-09 02:17:47,783 basehttp 3100 13992 "GET /about/ HTTP/1.1" 200 889
INFO 2024-06-09 02:17:47,961 basehttp 3100 13992 "GET /about/ HTTP/1.1" 200 889
INFO 2024-06-09 02:17:48,124 basehttp 3100 13992 "GET /about/ HTTP/1.1" 200 889
INFO 2024-06-09 02:17:48,307 basehttp 3100 13992 "GET /about/ HTTP/1.1" 200 889
INFO 2024-06-09 02:17:48,517 basehttp 3100 13992 "GET /about/ HTTP/1.1" 200 889
INFO 2024-06-09 02:17:59,455 autoreload 5420 11600 Watching for file changes with StatReloader
INFO 2024-06-09 02:18:02,169 basehttp 5420 9508 "GET /about/ HTTP/1.1" 200 889
INFO 2024-06-09 02:18:02,854 basehttp 5420 9508 "GET / HTTP/1.1" 200 947
INFO 2024-06-09 02:18:03,565 basehttp 5420 9508 "GET / HTTP/1.1" 200 947
INFO 2024-06-09 02:18:03,921 basehttp 5420 9508 "GET /about/ HTTP/1.1" 200 889
INFO 2024-06-09 02:18:04,389 basehttp 5420 9508 "GET / HTTP/1.1" 200 947
INFO 2024-06-09 02:18:04,833 basehttp 5420 9508 "GET / HTTP/1.1" 200 947
INFO 2024-06-09 02:18:05,165 basehttp 5420 9508 "GET /about/ HTTP/1.1" 200 889
INFO 2024-06-09 02:18:05,519 basehttp 5420 9508 "GET / HTTP/1.1" 200 947
INFO 2024-06-10 15:57:31,879 autoreload 15120 2280 Watching for file changes with StatReloader
INFO 2024-06-10 16:07:17,063 autoreload 11444 15024 Watching for file changes with StatReloader
INFO 2024-06-10 16:22:50,839 autoreload 11212 6320 Watching for file changes with StatReloader
INFO 2024-06-10 16:31:55,518 autoreload 1340 3760 Watching for file changes with StatReloader
INFO 2024-06-10 16:32:48,414 autoreload 1340 3760 C:\Users\User\Documents\Django\practika\myproject\myproject\settings.py changed, reloading.
INFO 2024-06-10 16:35:53,171 autoreload 14516 3308 Watching for file changes with StatReloader
INFO 2024-06-10 16:42:47,086 autoreload 14516 3308 C:\Users\User\Documents\Django\practika\myproject\myapp_lek1\apps.py changed, reloading.
INFO 2024-06-10 16:42:48,180 autoreload 4016 16112 Watching for file changes with StatReloader
INFO 2024-06-10 16:42:54,021 autoreload 16800 2972 Watching for file changes with StatReloader
INFO 2024-06-10 16:45:52,817 autoreload 16800 2972 C:\Users\User\Documents\Django\practika\myproject\myapp_sem1\views.py changed, reloading.
INFO 2024-06-10 16:45:53,993 autoreload 10440 1640 Watching for file changes with StatReloader
INFO 2024-06-10 16:48:39,551 autoreload 1464 12644 Watching for file changes with StatReloader
INFO 2024-06-10 16:50:03,486 autoreload 1464 12644 C:\Users\User\Documents\Django\practika\myproject\myapp_lek2\views.py changed, reloading.
INFO 2024-06-10 16:50:06,968 autoreload 8148 15796 Watching for file changes with StatReloader
WARNING 2024-06-10 16:50:10,798 log 8148 7208 Not Found: /
WARNING 2024-06-10 16:50:10,814 basehttp 8148 7208 "GET / HTTP/1.1" 404 2481
WARNING 2024-06-10 16:50:11,397 log 8148 7208 Not Found: /favicon.ico
WARNING 2024-06-10 16:50:11,397 basehttp 8148 7208 "GET /favicon.ico HTTP/1.1" 404 2532
INFO 2024-06-10 17:11:03,111 autoreload 13928 2780 Watching for file changes with StatReloader
INFO 2024-06-10 17:16:05,084 autoreload 15552 14808 Watching for file changes with StatReloader
INFO 2024-06-11 15:52:04,491 autoreload 16280 1852 Watching for file changes with StatReloader
WARNING 2024-06-11 15:52:17,931 log 16280 7500 Not Found: /
WARNING 2024-06-11 15:52:17,978 basehttp 16280 7500 "GET / HTTP/1.1" 404 2481
WARNING 2024-06-11 15:52:18,009 log 16280 7500 Not Found: /
WARNING 2024-06-11 15:52:18,009 basehttp 16280 7500 "GET / HTTP/1.1" 404 2481
WARNING 2024-06-11 15:52:18,475 log 16280 7500 Not Found: /favicon.ico
WARNING 2024-06-11 15:52:18,475 basehttp 16280 7500 "GET /favicon.ico HTTP/1.1" 404 2532
INFO 2024-06-11 15:52:30,974 basehttp 16280 14644 "GET /hw1 HTTP/1.1" 301 0
INFO 2024-06-11 15:52:30,998 basehttp 16280 15128 "GET /hw1/ HTTP/1.1" 200 3003
INFO 2024-06-11 15:52:33,613 basehttp 16280 16080 "GET /hw1/about/ HTTP/1.1" 200 3303
INFO 2024-06-11 15:52:34,618 basehttp 16280 16080 "GET /hw1/ HTTP/1.1" 200 3003
INFO 2024-06-11 15:52:35,184 basehttp 16280 16080 "GET /hw1/about/ HTTP/1.1" 200 3303
INFO 2024-06-11 15:59:19,514 autoreload 3304 8536 Watching for file changes with StatReloader
INFO 2024-06-11 15:59:27,105 basehttp 3304 6172 "GET /hw1/about/ HTTP/1.1" 200 3318
INFO 2024-06-11 15:59:27,981 basehttp 3304 6172 "GET /hw1/about/ HTTP/1.1" 200 3318
ERROR 2024-06-11 15:59:31,899 log 3304 5888 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 9, in index
    response = render_to_string('myapp_hw1/main1.html')
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: myapp_hw1/main1.html
ERROR 2024-06-11 15:59:31,901 basehttp 3304 5888 "GET /hw1/ HTTP/1.1" 500 73922
INFO 2024-06-11 16:00:08,727 autoreload 3304 8536 C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py changed, reloading.
INFO 2024-06-11 16:00:09,953 autoreload 8812 11380 Watching for file changes with StatReloader
INFO 2024-06-11 16:00:11,680 basehttp 8812 16080 "GET /hw1/ HTTP/1.1" 200 2990
INFO 2024-06-11 16:00:13,282 basehttp 8812 16080 "GET /lek2/lek2 HTTP/1.1" 200 11
INFO 2024-06-11 16:00:14,703 basehttp 8812 16080 "GET /lek2/lek2 HTTP/1.1" 200 11
INFO 2024-06-11 16:12:30,698 autoreload 8812 11380 C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py changed, reloading.
INFO 2024-06-11 16:12:32,008 autoreload 17260 12364 Watching for file changes with StatReloader
ERROR 2024-06-11 16:12:36,223 log 17260 9420 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 8, in index
    return render(request, 'myapp_hw1/index.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: myapp_hw1/index.html
ERROR 2024-06-11 16:12:36,226 basehttp 17260 9420 "GET /hw1/ HTTP/1.1" 500 78649
ERROR 2024-06-11 16:12:39,084 log 17260 9420 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 8, in index
    return render(request, 'myapp_hw1/index.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: myapp_hw1/index.html
ERROR 2024-06-11 16:12:39,084 basehttp 17260 9420 "GET /hw1/ HTTP/1.1" 500 78649
INFO 2024-06-11 16:17:37,665 autoreload 17260 12364 C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py changed, reloading.
INFO 2024-06-11 16:17:38,729 autoreload 11760 6440 Watching for file changes with StatReloader
INFO 2024-06-11 16:17:43,508 autoreload 4664 4916 Watching for file changes with StatReloader
WARNING 2024-06-11 16:17:44,859 log 4664 14600 Not Found: /
WARNING 2024-06-11 16:17:44,860 basehttp 4664 14600 "GET / HTTP/1.1" 404 2481
INFO 2024-06-11 16:17:48,579 basehttp 4664 14600 "GET /hw1/ HTTP/1.1" 200 2990
INFO 2024-06-13 10:12:20,148 autoreload 8172 11848 Watching for file changes with StatReloader
INFO 2024-06-13 10:12:22,904 basehttp 8172 13544 "GET /hw1/ HTTP/1.1" 200 2990
INFO 2024-06-13 10:12:24,073 basehttp 8172 14692 "GET /lek2/lek2 HTTP/1.1" 200 11
INFO 2024-06-13 10:12:27,127 basehttp 8172 14692 "GET /lek2/lek2 HTTP/1.1" 200 11
INFO 2024-06-13 10:13:20,643 basehttp 8172 8432 "GET /hw1/about/ HTTP/1.1" 200 3318
INFO 2024-06-13 10:13:23,704 basehttp 8172 8432 "GET /lek2/lek2 HTTP/1.1" 200 11
INFO 2024-06-13 10:13:46,599 basehttp 8172 15416 "GET /lek2/lek2 HTTP/1.1" 200 11
INFO 2024-06-13 10:13:48,196 basehttp 8172 15416 "GET /lek2/lek2 HTTP/1.1" 200 11
ERROR 2024-06-13 10:13:51,903 log 8172 8940 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 8, in index
    return render(request, 'myapp_hw1/main.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\defaulttags.py", line 479, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\resolvers.py", line 851, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for './about' not found. './about' is not a valid view function or pattern name.
ERROR 2024-06-13 10:13:51,909 basehttp 8172 8940 "GET /hw1/ HTTP/1.1" 500 130831
ERROR 2024-06-13 10:14:21,200 log 8172 8372 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 8, in index
    return render(request, 'myapp_hw1/main.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\defaulttags.py", line 479, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\resolvers.py", line 851, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'hw1/about' not found. 'hw1/about' is not a valid view function or pattern name.
ERROR 2024-06-13 10:14:21,206 basehttp 8172 8372 "GET /hw1/ HTTP/1.1" 500 130879
ERROR 2024-06-13 10:14:21,939 log 8172 8372 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 8, in index
    return render(request, 'myapp_hw1/main.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\defaulttags.py", line 479, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\resolvers.py", line 851, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'hw1/about' not found. 'hw1/about' is not a valid view function or pattern name.
ERROR 2024-06-13 10:14:21,943 basehttp 8172 8372 "GET /hw1/ HTTP/1.1" 500 130879
ERROR 2024-06-13 10:14:22,770 log 8172 8372 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 8, in index
    return render(request, 'myapp_hw1/main.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\defaulttags.py", line 479, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\resolvers.py", line 851, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'hw1/about' not found. 'hw1/about' is not a valid view function or pattern name.
ERROR 2024-06-13 10:14:22,776 basehttp 8172 8372 "GET /hw1/ HTTP/1.1" 500 130879
ERROR 2024-06-13 10:14:25,239 log 8172 11916 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 8, in index
    return render(request, 'myapp_hw1/main.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\defaulttags.py", line 479, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\resolvers.py", line 851, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'hw1/about' not found. 'hw1/about' is not a valid view function or pattern name.
ERROR 2024-06-13 10:14:25,244 basehttp 8172 11916 "GET /hw1/ HTTP/1.1" 500 130879
ERROR 2024-06-13 10:14:28,989 log 8172 11916 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 8, in index
    return render(request, 'myapp_hw1/main.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\defaulttags.py", line 479, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\resolvers.py", line 851, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'hw1/about' not found. 'hw1/about' is not a valid view function or pattern name.
ERROR 2024-06-13 10:14:28,996 basehttp 8172 11916 "GET /hw1/ HTTP/1.1" 500 130879
INFO 2024-06-13 10:14:44,493 basehttp 8172 16916 "GET /hw1/ HTTP/1.1" 200 2991
INFO 2024-06-13 10:14:45,630 basehttp 8172 16916 "GET /lek1/about HTTP/1.1" 200 27
INFO 2024-06-13 10:14:49,655 basehttp 8172 13736 "GET /lek1/about HTTP/1.1" 200 27
INFO 2024-06-13 10:14:53,210 basehttp 8172 13736 "GET /lek2/lek2 HTTP/1.1" 200 11
INFO 2024-06-13 10:16:17,440 basehttp 8172 13036 "GET /hw1/about/ HTTP/1.1" 200 3319
INFO 2024-06-13 10:16:24,140 basehttp 8172 13036 "GET /lek2/lek2 HTTP/1.1" 200 11
INFO 2024-06-13 10:16:25,747 basehttp 8172 14700 "GET /lek1/about HTTP/1.1" 200 27
INFO 2024-06-13 10:31:47,690 autoreload 16500 15684 Watching for file changes with StatReloader
WARNING 2024-06-13 10:31:50,290 log 16500 2888 Not Found: /
WARNING 2024-06-13 10:31:50,291 basehttp 16500 2888 "GET / HTTP/1.1" 404 2481
WARNING 2024-06-13 10:31:50,303 log 16500 2888 Not Found: /
WARNING 2024-06-13 10:31:50,304 basehttp 16500 2888 "GET / HTTP/1.1" 404 2481
WARNING 2024-06-13 10:31:50,432 log 16500 2888 Not Found: /favicon.ico
WARNING 2024-06-13 10:31:50,434 basehttp 16500 2888 "GET /favicon.ico HTTP/1.1" 404 2532
WARNING 2024-06-13 10:31:50,468 log 16500 2888 Not Found: /favicon.ico
WARNING 2024-06-13 10:31:50,471 basehttp 16500 2888 "GET /favicon.ico HTTP/1.1" 404 2532
INFO 2024-06-13 10:31:59,565 basehttp 16500 5444 "GET /hw1 HTTP/1.1" 301 0
INFO 2024-06-13 10:31:59,617 basehttp 16500 13180 "GET /hw1/ HTTP/1.1" 200 2991
INFO 2024-06-13 10:32:00,837 basehttp 16500 13180 "GET /lek1/about HTTP/1.1" 200 27
INFO 2024-06-13 10:32:03,396 basehttp 16500 9924 "GET /lek2/lek2 HTTP/1.1" 200 11
INFO 2024-06-13 10:32:12,731 basehttp 16500 9924 "GET /lek1/about HTTP/1.1" 200 27
INFO 2024-06-13 10:32:59,144 autoreload 16500 15684 C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py changed, reloading.
INFO 2024-06-13 10:33:00,310 autoreload 8796 13768 Watching for file changes with StatReloader
INFO 2024-06-13 10:33:35,125 autoreload 8796 13768 C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\urls.py changed, reloading.
INFO 2024-06-13 10:33:36,318 autoreload 15184 12564 Watching for file changes with StatReloader
INFO 2024-06-13 10:33:39,768 basehttp 15184 12932 "GET /hw1/ HTTP/1.1" 200 2992
INFO 2024-06-13 10:33:41,181 basehttp 15184 12932 "GET /hw1/about1/ HTTP/1.1" 200 3320
INFO 2024-06-13 10:33:42,953 basehttp 15184 12932 "GET /lek2/lek2 HTTP/1.1" 200 11
INFO 2024-06-13 10:33:44,766 basehttp 15184 16080 "GET /hw1/about1/ HTTP/1.1" 200 3320
INFO 2024-06-13 10:33:46,953 basehttp 15184 16080 "GET /hw1/about1/ HTTP/1.1" 200 3320
INFO 2024-06-13 10:33:54,857 autoreload 15184 12564 C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\urls.py changed, reloading.
INFO 2024-06-13 10:33:56,253 autoreload 14204 15188 Watching for file changes with StatReloader
INFO 2024-06-13 10:34:07,112 autoreload 14204 15188 C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py changed, reloading.
INFO 2024-06-13 10:34:08,743 autoreload 8392 16412 Watching for file changes with StatReloader
WARNING 2024-06-13 10:37:16,460 log 8392 11676 Not Found: /hw1/about1/
WARNING 2024-06-13 10:37:16,462 basehttp 8392 11676 "GET /hw1/about1/ HTTP/1.1" 404 2764
WARNING 2024-06-13 10:37:17,583 log 8392 11676 Not Found: /hw1/about1/
WARNING 2024-06-13 10:37:17,584 basehttp 8392 11676 "GET /hw1/about1/ HTTP/1.1" 404 2764
WARNING 2024-06-13 10:37:20,992 log 8392 11676 Not Found: /hw1/about1/
WARNING 2024-06-13 10:37:20,993 basehttp 8392 11676 "GET /hw1/about1/ HTTP/1.1" 404 2764
WARNING 2024-06-13 10:37:25,271 log 8392 15052 Not Found: /hw1/about1/
WARNING 2024-06-13 10:37:25,272 basehttp 8392 15052 "GET /hw1/about1/ HTTP/1.1" 404 2764
ERROR 2024-06-13 10:37:28,434 log 8392 15052 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 8, in index
    return render(request, 'myapp_hw1/main.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\defaulttags.py", line 479, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\resolvers.py", line 851, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'about/' not found. 'about/' is not a valid view function or pattern name.
ERROR 2024-06-13 10:37:28,445 basehttp 8392 15052 "GET /hw1/ HTTP/1.1" 500 130807
INFO 2024-06-13 10:37:58,111 basehttp 8392 15264 "GET /hw1/ HTTP/1.1" 200 2991
INFO 2024-06-13 10:37:59,319 basehttp 8392 15264 "GET /lek1/about HTTP/1.1" 200 27
ERROR 2024-06-13 10:44:57,517 log 8392 12896 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 8, in index
    return render(request, 'myapp_hw1/main.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\defaulttags.py", line 479, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\resolvers.py", line 851, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'myapp_hw1/about' not found. 'myapp_hw1/about' is not a valid view function or pattern name.
ERROR 2024-06-13 10:44:57,522 basehttp 8392 12896 "GET /hw1/ HTTP/1.1" 500 131023
ERROR 2024-06-13 10:44:58,890 log 8392 12896 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 8, in index
    return render(request, 'myapp_hw1/main.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\defaulttags.py", line 479, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\resolvers.py", line 851, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'myapp_hw1/about' not found. 'myapp_hw1/about' is not a valid view function or pattern name.
ERROR 2024-06-13 10:44:58,891 basehttp 8392 12896 "GET /hw1/ HTTP/1.1" 500 131023
INFO 2024-06-13 10:45:58,646 autoreload 6044 15936 Watching for file changes with StatReloader
ERROR 2024-06-13 10:46:02,113 log 6044 8788 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 8, in index
    return render(request, 'myapp_hw1/main.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\defaulttags.py", line 479, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\resolvers.py", line 851, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'data' not found. 'data' is not a valid view function or pattern name.
ERROR 2024-06-13 10:46:02,118 basehttp 6044 8788 "GET /hw1/ HTTP/1.1" 500 130759
ERROR 2024-06-13 10:46:03,219 log 6044 8788 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 8, in index
    return render(request, 'myapp_hw1/main.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\defaulttags.py", line 479, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\resolvers.py", line 851, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'data' not found. 'data' is not a valid view function or pattern name.
ERROR 2024-06-13 10:46:03,223 basehttp 6044 8788 "GET /hw1/ HTTP/1.1" 500 130759
ERROR 2024-06-13 10:46:07,187 log 6044 8788 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 8, in index
    return render(request, 'myapp_hw1/main.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\defaulttags.py", line 479, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\resolvers.py", line 851, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'data' not found. 'data' is not a valid view function or pattern name.
ERROR 2024-06-13 10:46:07,187 basehttp 6044 8788 "GET /hw1/ HTTP/1.1" 500 130759
WARNING 2024-06-13 10:46:09,002 log 6044 8788 Not Found: /
WARNING 2024-06-13 10:46:09,003 basehttp 6044 8788 "GET / HTTP/1.1" 404 2481
INFO 2024-06-13 10:49:56,717 basehttp 6044 8564 "GET /hw1/about HTTP/1.1" 301 0
INFO 2024-06-13 10:49:56,728 basehttp 6044 10436 "GET /hw1/about/ HTTP/1.1" 200 3319
INFO 2024-06-13 10:54:43,277 basehttp 6044 13936 "GET /hw1/about/ HTTP/1.1" 200 3319
INFO 2024-06-13 10:54:44,579 basehttp 6044 13936 "GET /lek1/about HTTP/1.1" 200 27
INFO 2024-06-13 10:54:47,152 basehttp 6044 13936 "GET /lek2/lek2 HTTP/1.1" 200 11
INFO 2024-06-13 11:00:02,683 autoreload 14480 17400 Watching for file changes with StatReloader
INFO 2024-06-13 11:00:06,411 basehttp 14480 12708 "GET /lek1/about HTTP/1.1" 200 27
ERROR 2024-06-13 11:00:09,012 log 14480 12708 Internal Server Error: /hw1/about/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 13, in about
    return render(request, 'myapp_hw1/data.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 134, in render
    compiled_parent = self.get_parent(context)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 131, in get_parent
    return self.find_template(parent, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 109, in find_template
    template, origin = context.template.engine.find_template(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 565, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 18: 'static'. Did you forget to register or load this tag?
ERROR 2024-06-13 11:00:09,012 basehttp 14480 12708 "GET /hw1/about/ HTTP/1.1" 500 156435
INFO 2024-06-13 11:00:58,794 autoreload 7900 14372 Watching for file changes with StatReloader
ERROR 2024-06-13 11:01:02,434 log 7900 12372 Internal Server Error: /hw1/about/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 13, in about
    return render(request, 'myapp_hw1/data.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 134, in render
    compiled_parent = self.get_parent(context)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 131, in get_parent
    return self.find_template(parent, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 109, in find_template
    template, origin = context.template.engine.find_template(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 565, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 18: 'static'. Did you forget to register or load this tag?
ERROR 2024-06-13 11:01:02,441 basehttp 7900 12372 "GET /hw1/about/ HTTP/1.1" 500 156419
ERROR 2024-06-13 11:01:05,051 log 7900 12372 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 8, in index
    return render(request, 'myapp_hw1/main.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 134, in render
    compiled_parent = self.get_parent(context)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 131, in get_parent
    return self.find_template(parent, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 109, in find_template
    template, origin = context.template.engine.find_template(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 565, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 18: 'static'. Did you forget to register or load this tag?
ERROR 2024-06-13 11:01:05,071 basehttp 7900 12372 "GET /hw1/ HTTP/1.1" 500 157131
ERROR 2024-06-13 11:01:06,145 log 7900 12372 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 8, in index
    return render(request, 'myapp_hw1/main.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 134, in render
    compiled_parent = self.get_parent(context)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 131, in get_parent
    return self.find_template(parent, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 109, in find_template
    template, origin = context.template.engine.find_template(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 565, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 18: 'static'. Did you forget to register or load this tag?
ERROR 2024-06-13 11:01:06,150 basehttp 7900 12372 "GET /hw1/ HTTP/1.1" 500 157131
ERROR 2024-06-13 11:01:06,720 log 7900 12372 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 502, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 8, in index
    return render(request, 'myapp_hw1/main.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 134, in render
    compiled_parent = self.get_parent(context)
                      ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 131, in get_parent
    return self.find_template(parent, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 109, in find_template
    template, origin = context.template.engine.find_template(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    return parser.parse()
           ^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 504, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 565, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 18: 'static'. Did you forget to register or load this tag?
ERROR 2024-06-13 11:01:06,720 basehttp 7900 12372 "GET /hw1/ HTTP/1.1" 500 157131
ERROR 2024-06-13 11:02:53,398 log 7900 11284 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 8, in index
    return render(request, 'myapp_hw1/main.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\defaulttags.py", line 479, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\resolvers.py", line 851, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for '' not found. '' is not a valid view function or pattern name.
ERROR 2024-06-13 11:02:53,398 basehttp 7900 11284 "GET /hw1/ HTTP/1.1" 500 130947
ERROR 2024-06-13 11:02:54,484 log 7900 11284 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 8, in index
    return render(request, 'myapp_hw1/main.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\defaulttags.py", line 479, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\resolvers.py", line 851, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for '' not found. '' is not a valid view function or pattern name.
ERROR 2024-06-13 11:02:54,489 basehttp 7900 11284 "GET /hw1/ HTTP/1.1" 500 130947
INFO 2024-06-13 11:03:17,503 basehttp 7900 15700 "GET /hw1/ HTTP/1.1" 200 2991
INFO 2024-06-13 11:03:18,791 basehttp 7900 15700 "GET /lek1/about HTTP/1.1" 200 27
INFO 2024-06-13 11:04:27,729 autoreload 17252 13544 Watching for file changes with StatReloader
INFO 2024-06-13 11:04:31,011 basehttp 17252 16024 "GET /hw1/ HTTP/1.1" 200 2996
INFO 2024-06-13 11:04:31,513 basehttp 17252 16024 "GET /hw1/ HTTP/1.1" 200 2996
INFO 2024-06-13 11:04:31,802 basehttp 17252 16024 "GET /hw1/ HTTP/1.1" 200 2996
INFO 2024-06-13 11:04:32,287 basehttp 17252 16024 "GET /lek1/about HTTP/1.1" 200 27
INFO 2024-06-13 11:05:22,415 autoreload 13424 5844 Watching for file changes with StatReloader
INFO 2024-06-13 11:05:29,678 basehttp 13424 2976 "GET /hw1/ HTTP/1.1" 200 3002
INFO 2024-06-13 11:05:30,229 basehttp 13424 2976 "GET /hw1/ HTTP/1.1" 200 3002
WARNING 2024-06-13 11:05:30,787 basehttp 13424 2976 "GET /static/myapp_hw1 HTTP/1.1" 404 1786
INFO 2024-06-13 11:05:57,835 basehttp 13424 17108 "GET /hw1/ HTTP/1.1" 200 3012
WARNING 2024-06-13 11:05:58,572 basehttp 13424 17108 "GET /static/myapp_hw1/data.html HTTP/1.1" 404 1816
INFO 2024-06-13 11:06:15,606 basehttp 13424 8460 "GET /hw1/ HTTP/1.1" 200 3022
WARNING 2024-06-13 11:06:16,296 basehttp 13424 8460 "GET /static/myapp_hw1/templates/data.html HTTP/1.1" 404 1846
INFO 2024-06-13 11:07:02,550 basehttp 13424 7632 "GET /hw1/ HTTP/1.1" 200 2991
INFO 2024-06-13 11:07:03,096 basehttp 13424 7632 "GET /hw1/ HTTP/1.1" 200 2991
INFO 2024-06-13 11:07:03,549 basehttp 13424 7632 "GET /lek1/about HTTP/1.1" 200 27
INFO 2024-06-13 13:54:48,903 autoreload 10948 16152 Watching for file changes with StatReloader
INFO 2024-06-13 13:54:54,282 basehttp 10948 15740 "GET /hw1/ HTTP/1.1" 200 2991
INFO 2024-06-13 13:54:54,873 basehttp 10948 15740 "GET /hw1/ HTTP/1.1" 200 2991
INFO 2024-06-13 13:54:55,514 basehttp 10948 15740 "GET /lek1/about HTTP/1.1" 200 27
INFO 2024-06-13 13:54:57,984 basehttp 10948 15740 "GET /lek2/lek2 HTTP/1.1" 200 11
INFO 2024-06-13 13:54:59,295 basehttp 10948 15740 "GET /lek1/about HTTP/1.1" 200 27
INFO 2024-06-13 13:55:01,959 basehttp 10948 15740 "GET /lek1/about HTTP/1.1" 200 27
INFO 2024-06-13 13:55:14,637 autoreload 9472 748 Watching for file changes with StatReloader
INFO 2024-06-13 13:55:48,637 autoreload 10524 11192 Watching for file changes with StatReloader
INFO 2024-06-13 13:55:55,003 basehttp 10524 8204 "GET /hw1/ HTTP/1.1" 200 2991
INFO 2024-06-13 13:55:56,442 basehttp 10524 8204 "GET /lek1/about HTTP/1.1" 200 27
INFO 2024-06-13 13:58:48,050 basehttp 10524 18912 "GET /hw1/ HTTP/1.1" 200 2991
INFO 2024-06-13 13:58:48,855 basehttp 10524 18912 "GET /hw1/ HTTP/1.1" 200 2991
INFO 2024-06-13 13:58:49,390 basehttp 10524 18912 "GET /hw1/about/ HTTP/1.1" 200 3319
INFO 2024-06-13 13:58:52,049 basehttp 10524 18912 "GET /hw1/about/ HTTP/1.1" 200 3319
INFO 2024-06-13 13:59:03,102 autoreload 10524 11192 C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\urls.py changed, reloading.
INFO 2024-06-13 13:59:04,229 autoreload 9084 15112 Watching for file changes with StatReloader
ERROR 2024-06-13 13:59:07,369 log 9084 7744 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\base.py", line 71, in reverse
    extra, resolver = resolver.namespace_dict[ns]
                      ~~~~~~~~~~~~~~~~~~~~~~~^^^^
KeyError: 'myapp_hw1'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 8, in index
    return render(request, 'myapp_hw1/main.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\defaulttags.py", line 479, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\base.py", line 82, in reverse
    raise NoReverseMatch("%s is not a registered namespace" % key)
django.urls.exceptions.NoReverseMatch: 'myapp_hw1' is not a registered namespace
ERROR 2024-06-13 13:59:07,376 basehttp 9084 7744 "GET /hw1/ HTTP/1.1" 500 133172
ERROR 2024-06-13 13:59:11,952 log 9084 7744 Internal Server Error: /hw1/
Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\base.py", line 71, in reverse
    extra, resolver = resolver.namespace_dict[ns]
                      ~~~~~~~~~~~~~~~~~~~~~~~^^^^
KeyError: 'myapp_hw1'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\views.py", line 8, in index
    return render(request, 'myapp_hw1/main.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\loader_tags.py", line 159, in render
    return compiled_parent._render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 1000, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\base.py", line 961, in render_annotated
    return self.render(context)
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\template\defaulttags.py", line 479, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\Documents\Django\practika\.venv\Lib\site-packages\django\urls\base.py", line 82, in reverse
    raise NoReverseMatch("%s is not a registered namespace" % key)
django.urls.exceptions.NoReverseMatch: 'myapp_hw1' is not a registered namespace
ERROR 2024-06-13 13:59:11,957 basehttp 9084 7744 "GET /hw1/ HTTP/1.1" 500 133172
INFO 2024-06-13 13:59:23,840 autoreload 9084 15112 C:\Users\User\Documents\Django\practika\myproject\myapp_hw1\urls.py changed, reloading.
INFO 2024-06-13 13:59:24,946 autoreload 19008 10696 Watching for file changes with StatReloader
INFO 2024-06-13 13:59:31,680 basehttp 19008 12444 "GET /hw1/ HTTP/1.1" 200 2991
INFO 2024-06-13 13:59:32,953 basehttp 19008 12444 "GET /hw1/about/ HTTP/1.1" 200 3319
INFO 2024-06-13 14:02:09,606 basehttp 19008 14864 "GET /hw1/about/ HTTP/1.1" 200 3314
INFO 2024-06-13 14:02:10,763 basehttp 19008 14864 "GET /hw1/ HTTP/1.1" 200 2986
INFO 2024-06-13 14:02:12,081 basehttp 19008 14864 "GET /hw1/about/ HTTP/1.1" 200 3314
INFO 2024-06-13 14:02:12,919 basehttp 19008 14864 "GET /hw1/ HTTP/1.1" 200 2986
INFO 2024-06-13 14:02:13,588 basehttp 19008 18444 "GET /hw1/about/ HTTP/1.1" 200 3314
INFO 2024-06-13 14:02:14,083 basehttp 19008 18444 "GET /hw1/ HTTP/1.1" 200 2986
INFO 2024-06-13 14:02:14,583 basehttp 19008 18444 "GET /hw1/about/ HTTP/1.1" 200 3314
INFO 2024-06-13 14:02:15,103 basehttp 19008 18444 "GET /hw1/ HTTP/1.1" 200 2986
INFO 2024-06-13 14:02:15,564 basehttp 19008 18444 "GET /hw1/about/ HTTP/1.1" 200 3314
INFO 2024-06-13 21:30:37,096 autoreload 2572 16004 Watching for file changes with StatReloader
INFO 2024-06-13 21:33:36,240 autoreload 15768 15756 Watching for file changes with StatReloader
INFO 2024-06-13 21:34:26,563 autoreload 12664 20132 Watching for file changes with StatReloader
INFO 2024-06-13 21:35:52,494 autoreload 21768 21052 Watching for file changes with StatReloader
INFO 2024-06-13 21:36:20,598 autoreload 21768 21052 C:\GB\Django\practika_new2\myproject\myapp_hw2\views.py changed, reloading.
INFO 2024-06-13 21:36:21,072 autoreload 21836 22316 Watching for file changes with StatReloader
INFO 2024-06-13 21:36:59,666 autoreload 1580 19688 Watching for file changes with StatReloader
INFO 2024-06-13 21:37:27,161 autoreload 3904 20012 Watching for file changes with StatReloader
INFO 2024-06-13 21:37:52,059 autoreload 18156 11108 Watching for file changes with StatReloader
INFO 2024-06-13 21:38:02,844 autoreload 16124 7468 Watching for file changes with StatReloader
WARNING 2024-06-13 21:38:04,942 log 16124 7564 Not Found: /
WARNING 2024-06-13 21:38:04,943 basehttp 16124 7564 "GET / HTTP/1.1" 404 2585
WARNING 2024-06-13 21:38:05,108 log 16124 7564 Not Found: /favicon.ico
WARNING 2024-06-13 21:38:05,109 basehttp 16124 7564 "GET /favicon.ico HTTP/1.1" 404 2636
INFO 2024-06-13 21:38:12,300 basehttp 16124 7564 "GET /hw1 HTTP/1.1" 301 0
INFO 2024-06-13 21:38:12,323 basehttp 16124 17696 "GET /hw1/ HTTP/1.1" 200 2986
INFO 2024-06-13 21:38:14,040 basehttp 16124 17696 "GET /hw1/about/ HTTP/1.1" 200 3314
INFO 2024-06-13 21:38:15,656 basehttp 16124 17696 "GET /hw1/ HTTP/1.1" 200 2986
INFO 2024-06-13 21:38:18,690 basehttp 16124 17696 "GET /hw2/ HTTP/1.1" 200 2986
INFO 2024-06-13 21:38:20,121 basehttp 16124 17696 "GET /hw1/about/ HTTP/1.1" 200 3314
INFO 2024-06-13 21:38:20,876 basehttp 16124 17696 "GET /hw1/ HTTP/1.1" 200 2986
INFO 2024-06-13 21:38:21,496 basehttp 16124 17696 "GET /hw1/about/ HTTP/1.1" 200 3314
INFO 2024-06-13 21:38:21,911 basehttp 16124 17696 "GET /hw1/ HTTP/1.1" 200 2986
INFO 2024-06-13 21:38:22,002 basehttp 16124 17696 "GET /hw1/ HTTP/1.1" 200 2986
INFO 2024-06-13 21:38:22,551 basehttp 16124 17696 "GET /hw1/about/ HTTP/1.1" 200 3314
INFO 2024-06-13 21:38:23,266 basehttp 16124 17696 "GET /hw1/ HTTP/1.1" 200 2986
INFO 2024-06-13 21:38:23,358 basehttp 16124 17696 "GET /hw1/ HTTP/1.1" 200 2986
INFO 2024-06-13 21:38:23,795 basehttp 16124 17696 "GET /hw1/about/ HTTP/1.1" 200 3314
INFO 2024-06-13 21:38:23,886 basehttp 16124 17696 "GET /hw1/about/ HTTP/1.1" 200 3314
INFO 2024-06-13 21:38:24,373 basehttp 16124 17696 "GET /hw1/ HTTP/1.1" 200 2986
INFO 2024-06-13 21:40:05,330 autoreload 16124 7468 C:\GB\Django\practika_new2\myproject\myapp_hw2\models.py changed, reloading.
INFO 2024-06-13 21:40:05,831 autoreload 21448 15344 Watching for file changes with StatReloader
INFO 2024-06-13 21:40:44,468 autoreload 21448 15344 C:\GB\Django\practika_new2\myproject\myapp_hw2\models.py changed, reloading.
INFO 2024-06-13 21:40:45,013 autoreload 740 21748 Watching for file changes with StatReloader
INFO 2024-06-13 21:44:37,305 autoreload 740 21748 C:\GB\Django\practika_new2\myproject\myapp_hw2\models.py changed, reloading.
INFO 2024-06-13 21:44:37,826 autoreload 18728 21664 Watching for file changes with StatReloader
INFO 2024-06-13 22:26:03,740 autoreload 18728 21664 C:\GB\Django\practika_new2\myproject\myapp_hw2\models.py changed, reloading.
INFO 2024-06-13 22:26:04,406 autoreload 13604 6900 Watching for file changes with StatReloader
INFO 2024-06-13 22:27:28,530 autoreload 13604 6900 C:\GB\Django\practika_new2\myproject\myapp_hw2\models.py changed, reloading.
INFO 2024-06-13 22:27:29,022 autoreload 19872 21920 Watching for file changes with StatReloader
INFO 2024-06-13 22:29:40,422 autoreload 19872 21920 C:\GB\Django\practika_new2\myproject\myapp_hw2\models.py changed, reloading.
INFO 2024-06-13 22:29:40,934 autoreload 21244 984 Watching for file changes with StatReloader
INFO 2024-06-13 22:31:35,535 autoreload 21244 984 C:\GB\Django\practika_new2\myproject\myapp_hw2\models.py changed, reloading.
INFO 2024-06-13 22:31:36,037 autoreload 10488 12000 Watching for file changes with StatReloader
INFO 2024-06-13 22:32:10,169 autoreload 10488 12000 C:\GB\Django\practika_new2\myproject\myapp_hw2\models.py changed, reloading.
INFO 2024-06-13 22:32:10,724 autoreload 3400 2340 Watching for file changes with StatReloader
